logverification/validation_test.go

```diff
@@ -12,7 +13,7 @@ func TestVerifiableEvent_Validate(t *testing.T) {
 	type fields struct {
 		AppId           string
 		LogId           []byte
-		MMREntryFields  *MMREntryFields
+		MMREntryFields  *app.MMREntryFields
```

Comment:
Seems a bit strange to have public fields of a private struct?

Reply:
names such as appId usually have acronyms capitailsed viz: appID

Reply:
can make them private, its just a test struct :)

Reply:
done!